src/controllers/metadata.js

```diff
@@ -32,4 +10,29 @@ const metadata = {
  *
  * @returns {void}
  */
-exports.getResource = function * () { this.body = metadata }
+module.exports = (config) => ({
+  getResource: function * () {
+    const base = config.getIn(['server', 'base_uri'])
+    const metadata = {
```

Comment:
The problem was that before, metadata would only be re-generated once (when it was imported the first time). I can make it generate once each time that the module.exports is called.